app/javascript/components/track/approaches-elements/DiggingDeeper.tsx

```diff
@@ -22,15 +22,21 @@ export function DiggingDeeper({
 }): JSX.Element {
   return (
     <div className="mb-48">
-      <section className="shadow-lgZ1 !py-[18px] mb-16 rounded-8 px-20 lg:px-32 py-20 lg:py-24">
-        <h2 className="mb-8 text-h2">Digging deeper</h2>
-        <div
-          className="c-textual-content --small"
-          dangerouslySetInnerHTML={{ __html: introduction.html }}
-        />
-      </section>
+      {introduction.html.length > 0 ? (
+        <>
+          <section className="shadow-lgZ1 !py-[18px] mb-16 rounded-8 px-20 lg:px-32 py-20 lg:py-24">
```

Comment:
@dem4ron:
1. This needs a `bg-white` to look right. 
2. I don't like the important flag here - why do we have that? 
3. We should stick to the base we're using, so make this `py-20` and shout at @taiyab about it :)